doc/configuration/statistic.md

```diff
@@ -70,7 +72,18 @@ classifier "bayes" {
 .include(try=true; priority=10) "$LOCAL_CONFDIR/override.d/statistic.conf"
 ~~~
 
-To enable per-user statistics, you can add the `users_enabled = true` property to the configuration of the classifier. However, it is important to ensure that Rspamd is called at the final delivery stage (e.g., LDA mode) to avoid issues with multi-recipient messages. When dealing with multi-recipient messages, Rspamd will use the first recipient for user-based statistics. 
+You are also recommended to use [`bayes_expiry` module](https://rspamd.com/doc/modules/bayes_expiry.html) to maintain your statistics database.
+
+Please note that `classifier-bayes.conf` is child config of `statistics.conf` which created for simplicity, you should not use them both at once.
```

Comment:
Should be `statistic.conf` (singular).

`statistic.conf` includes are quite irregular, I'm doubtful why should we have those really, they are almost or exactly the same as `rspamd.conf.local` and `rspamd.conf.override` respectively; they are likely better unused.

Reply:
What you mean by:

> `statistic.conf` includes are quite irregular, I'm doubtful why should we have those really, they are almost or exactly the same as `rspamd.conf.local` and `rspamd.conf.override` respectively; they are likely better unused.

I don't get

Reply:
I disagree. It's very handy in some cases. Let assume you need to configure [this](https://rspamd.com/doc/modules/bayes_expiry.html#limiting-memory-usage-to-a-fixed-amount). If you would use `local.d/statistic.conf` instead then you couldn't write just five lines, you have to copy everything from statistic.conf as well.